pkg/controller/storagecluster/modify.go

```diff
@@ -0,0 +1,18 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// StatusUpdate updates verbose log to a shorter and concise log
+func (r *ReconcileStorageCluster) StatusUpdate(obj runtime.Object) error {
+	err := r.client.Status().Update(context.TODO(), obj)
+	if errors.IsConflict(err) {
+		return fmt.Errorf("the object has been updated")
```

Comment:
```suggestion
		return fmt.Errorf("the object has been modified")
```

Reply:
Yes, was planning to add test cases. I can think of only 2. What's the third one?.

-An Object is updated
- An object is not updated
- what about third?

Reply:
1. object has not been modified
2. object has been modified
3. object doesn't exist

Reply:
done!